configure.in

```diff
@@ -396,6 +413,17 @@ if test "$NEED_MAKE_MOD2XMOD" = $TRUE; then
     AC_SUBST(MPFR_LIB_DIR)
 fi
 
+else # with_native_fortran_compiler != yes
+
+NEED_MAKE_MOD2XMOD=$FALSE
+KIND_FORTRAN=none
+MOD2XMOD=$FALSE
+
+AC_SUBST(MOD2XMOD)
```

Comment:
Why not using AC_SUBST(MODXMOD, [$FALSE]) ? Same question for KIND_FORTRAN & NEED_MAKE_MOD2XMOD

Reply:
When making changes, I tried to keep it as close as possible to original, which used 2 different lines for variable assignment and substitution in make file.